cob_bringup/drivers/bms.launch

```diff
@@ -5,11 +5,11 @@
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="sim" default="false"/>
 
-	<node unless="$(arg sim)" name="bms" pkg="cob_bms_driver" type="bms_driver_node" output="screen" clear_params="true">
-		<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/bms.yaml"/>
-	</node>
+	<node unless="$(arg sim)" name="bms" pkg="cob_bms_driver" type="bms_driver_node" output="screen" clear_params="true" />
+	<node if="$(arg sim)" ns="bms" name="fake_bms" pkg="cob_bms_driver" type="fake_bms.py" output="screen" />
+	<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/bms.yaml" ns="bms"/>
```

Comment:
Are you sure this works?
The `bms_driver_node` expects the parameters to be private...thus, I'd like to keep
```
<node unless="$(arg sim)" name="bms" pkg="cob_bms_driver" type="bms_driver_node" output="screen" clear_params="true">
	<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/bms.yaml"/>
</node>
```